fees/ether-fi/index.ts

```diff
@@ -0,0 +1,49 @@
+// https://etherfi.gitbook.io/etherfi/liquid/technical-documentation#fees
+import * as sdk from "@defillama/sdk";
+import { Adapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+
+const LIQUID_VAULT = "0xeA1A6307D9b18F8d1cbf1c3Dd6aad8416C06a221";
+const YEAR = 365;
+const ANNUAL_FEE_RATE = 0.02;
+
+const getDailyFees = async (api: sdk.ChainApi): Promise<number> => {
+  const dailyFeeRate = ANNUAL_FEE_RATE / YEAR;
+  const [asset, totalSupply] = await Promise.all([
+    await api.call({
+      target: LIQUID_VAULT,
+      abi: "function asset() external view returns (address)",
+    }),
+    await api.call({
+      target: LIQUID_VAULT,
+      abi: "function totalAssets() external view returns (uint256 assets)",
+    }),
+  ]);
+
+  api.addTokens(asset, totalSupply);
+  return (await api.getUSDValue()) * dailyFeeRate;
+};
+
+const adapter: Adapter = {
+  version: 2,
+  adapter: {
+    [CHAIN.ETHEREUM]: {
+      fetch: async () => {
+        const api = new sdk.ChainApi({ chain: CHAIN.ETHEREUM });
```

Comment:
function `sdk.ChainApi` can't support backfill data, could you change use parma `options: FetchOptions`

and use ` options.api.call`
and use `  dailyFees.add(asset, totalSupply)` instend `api.addTokens`